pulsar-client/src/main/java/org/apache/pulsar/client/impl/HttpLookupService.java

```diff
@@ -155,6 +156,30 @@ public String getServiceUrl() {
         return future;
     }
 
+    @Override
+    public CompletableFuture<Optional<SchemaInfo>> getSchema(TopicName topicName, byte[] version) {
```

Comment:
Can we collapse the above `getSchema(TopicName topicName)` into the new method passing a null version ?

Reply:
👍
I'm already refactor get schema in HttpLookupService and BinaryProtoLookupService.